src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -6214,6 +6214,9 @@ public void M()
             // (8,25): error CS0102: The type 'C' already contains a definition for 'Test'
             //     public required int Test;
             Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "Test").WithArguments("C", "Test").WithLocation(8, 25),
+            // (12,19): error CS9035: Required member 'C.Test' must be set in the object initializer or attribute constructor.
+            //         C c = new C { T = 42 };
+            Diagnostic(ErrorCode.ERR_RequiredMemberMustBeSet, "C").WithArguments("C.Test").WithLocation(12, 19),
```

Comment:
Previously, this type would be marked as having a malformed required members data, as we checked the builder for the current type when looking for duplicates, rather than looking at the base. I could add back in an additional check for current type duplicates, but I don't think that's necessary: it's a source error anyway, and there is a good likelihood that the user wanted one of the required properties anyway, so it seems like a fine error to report.